test/hotspot/jtreg/testlibrary_tests/template_framework/tests/TestTemplate.java

```diff
@@ -403,18 +466,23 @@ public static void testHookSimple() {
     public static void testHookIsAnchored() {
         var hook1 = new Hook("Hook1");
 
-        var template0 = Template.make(() -> body("isAnchored: ", hook1.isAnchored(), "\n"));
+        var template0 = Template.make(() -> scope("t0 isAnchored: ", hook1.isAnchored(a -> scope(a)), "\n"));
```

Comment:
You could directly use `Template::scope` instead of `a -> scope(a)`. Same below.

Reply:
But is that really more readable? I fear not really. And it is also longer... I'm a little torn with these. @robcasloz @mhaessig what are your votes?

Before:
`"t2 isAnchored: ", hook1.isAnchored(a -> scope(a)), "\n",`
After:
`"t2 isAnchored: ", hook1.isAnchored(Template::scope), "\n",`